metrics/mixin/alerts/perf.libsonnet

```diff
@@ -1,26 +1,43 @@
 {
   prometheusAlerts+:: {
-    groups+: [
-      {
-        name: 'mds-performance-alerts.rules',
-        rules: [
-          {
-            alert: 'MDSCacheUsageHigh',
-            expr: |||
-              ceph_mds_mem_rss / on(ceph_daemon) group_left(job)(label_replace(kube_pod_container_resource_requests{container="mds", resource="memory"}, "ceph_daemon", "mds.$1", "pod", "rook-ceph-mds-(.*)-(.*)") * .5) > .95
-            ||| % $._config,
-            'for': $._config.mdsCacheUsageAlertTime,
-            labels: {
-              severity: 'critical',
+    groups+:
+      [
+        {
+          name: 'mds-performance-alerts.rules',
+          rules: [
+            {
+              alert: 'MDSCacheUsageHigh',
+              expr: |||
```

Comment:
Is this formatting change caused by a linter? Doesn't look like it's needed here.

Reply:
Happened while addressing merge conflicts. Fixed it. Thanks.